src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -62,14 +64,15 @@ std::string ov::npuw::util::fmt(std::size_t number, std::size_t total) {
 }
 
 namespace {
-
+#if defined(HAVE_AVX2)
```

Comment:
who defines `HAVE_AVX2` for this file if AVX2 is enabled in cmake?
In CPU source files, we use `cross_compiled_file` which defines `HAVE_<instruction set>` automatically based on compiled architectures. Looks like in case of NPU this code will always be disabled, right?

Reply:
it seems if that code worked for @eshiryae, somehow it is defined when `util.cpp` is compiled - otherwise an exception would be thrown when running a model with DCOFF configured. Anyways let's wait for Ekaterina's reply

Reply:
Added `cross_compiled_file` using for AVX2 specific code. 
Yes, basically in previous implementation this code was always disabled and exception was thrown when model is running. Now  HAVE_AVX2 should be enabled correctly, but see other behavior - failed with exception, even though file were compiled with avx2 flags. @ilya-lavrenov could you please help to understand what is missing?

Reply:
Found that my Linux VM doesn't have AVX2 support, checked on other client - everything is working.